docker/docker-compose.yml

```diff
@@ -15,32 +15,31 @@ services:
       - zookeeper:/var/lib/zookeeper/data
 
   kafka1:
-    image: confluentinc/cp-kafka:5.3.0
+    image: confluentinc/cp-kafka:latest
```

Comment:
You need to be careful with this, usually you always want to specify the version you use. Otherwise things can all of a sudden break without you having changed anything.